### ISSUE TITLE
QR not generating when number is large

### ISSUE THREAD

**Issue by vaibhava17** [Shudarshan/QR-code-generator#14]
I will add a specific limit to the number field with error message on submitting. Which will help in handling this issue.

<img width="1140" alt="image" src="https://github.com/mrsudarshanrai/QR-code-generator/assets/65801959/f9026728-90ef-4521-8e11-5368872df8f7">

**Comment by mrsudarshanrai** [comment]
@vaibhava17 You can work on it.

**Comment by Ashish89088** [comment]
Is this issue still open, I want to work on this

**Comment by Ashish89088** [comment]
Hi I have Fix this Issue and send you the pull request. please review my code and merge my pull request.

**Comment by vaibhava17** [comment]
@Ashish89088 i have already done that and they are not active

**Comment by Ashish89088** [comment]
Yes I have seen that you have taken mod with 400 , but this code is running fine for all inputs. Your case fails when user enters suppose 401 , so  401%400 = 1 , and for this size the qr is not showing. I have bounded the range of input in the range of [200,400] .

**Comment by Ashish89088** [comment]
I am attaching the screenshots for two different input. The QR is now shown for all types of input. Please review this and run it for different testcases once.
![Screenshot (682)](https://github.com/mrsudarshanrai/QR-code-generator/assets/109150643/20eec657-437a-460a-965b-abf8af883792)
![Screenshot (681)](https://github.com/mrsudarshanrai/QR-code-generator/assets/109150643/21546bf7-0231-4e3e-a220-9f387b46aea1)
